umbraco/Umbraco-CMS
# V8/Warn editor when deleting a content item that is in use

### Prerequisites


If there's an existing issue for this PR then this fixes #8694 

### Description
This PR introduces a warning to editors when they try to delete a cotent item when the item or one of the underlying items of a items is in use.

### Note for reviewers

This PR builds on top of PR #9118  so it's recommend that that one is reviewed and merged first.

### Steps to test

See steps for content in #9116. When you try to delete the item a warning is shown + information of the references.
